resources/guis.yml

```diff
@@ -233,35 +182,18 @@ guis:
   image_tag: guis/pocketgit@2x.png
   platforms:
     - Android
-  price: 1.99€
+  price: €2.49/$1.99
   license: Proprietary
   order: 21
 - name: GitDrive
   url: http://gitdrive.com/
   image_tag: guis/gitdrive@2x.png
   platforms:
     - iOS
-  price: Free with in-app purchases
+  # No details what the $6.99 in-app purchase does, all it says is "Unlimited Version"
+  price: Free / $6.99
   license: Proprietary
   order: 37
-- name: GitX-dev
-  url: https://rowanj.github.io/gitx/
-  image_tag: guis/gitx@2x.png
-  platforms:
-  - Mac
-  price: Free
-  license: GNU GPL
-  order: 33
-- name: GitBlade
-  url: https://gitblade.com
-  image_tag: guis/gitblade@2x.png
-  platforms:
-    - Linux
-    - Mac
-    - Windows
-  price: Free Lite version, $59.99/user/year for PRO version
-  license: Proprietary
-  order: 34
```

Comment:
For convenience, my reasoning for this was:

> not updated since 2020 https://gitblade.com/releases

Which on second thought might not be that fair, so this is a good example for you to say yay/nay to a removal.